vcf2fhir/test/test_integration.py

```diff
@@ -333,19 +341,21 @@ def test_multiple_region_studied(self):
             region_studied_filename=region_studied_filename,
             nocall_filename=nocall_filename)
         o_vcf_2_fhir.convert(output_filename)
-        actual_fhir_json = json.load(open(output_filename))
-        # Validate the pased sequence relationship
-        self.assertEqual(_validate_phase_rel(
-            actual_fhir_json, {31: [24, 25], 32: [25, 26]}), True)
-        # Validate: list of observation uids and list of result uids same.
-        # Also set the uids to '' to avoid guid comparison in next step
-        map_ids = _get_uids_map(actual_fhir_json)
-        self.assertEqual(map_ids['obv_ids'], map_ids['result_ids'])
-        actual_fhir_json['issued'] = ''
-        # Finally, check if the acutal json after removing all uids is same as
-        # exppected json
-        expected_fhir_json = json.load(open(expected_outfult_filename))
-        self.assertEqual(actual_fhir_json, expected_fhir_json)
+        with open(output_filename) as output_file,\
+                open(expected_output_filename) as expected_output_file:
+            actual_fhir_json = json.load(output_file)
+            # Validate the pased sequence relationship
+            self.assertEqual(_validate_phase_rel(
+                actual_fhir_json, {31: [24, 25], 32: [25, 26]}), True)
+            # Validate: list of observation uids and list of result uids same.
+            # Also set the uids to '' to avoid guid comparison in next step
+            map_ids = _get_uids_map(actual_fhir_json)
+            self.assertEqual(map_ids['obv_ids'], map_ids['result_ids'])
+            actual_fhir_json['issued'] = ''
+            # Finally, check if the acutal json after removing all uids is
+            # same as exppected json
```

Comment:
nit: typo from before. Fix in all other test too.
```suggestion
            # same as expected json
```